jobs/aadAppRegistration.yml

```diff
@@ -1,38 +1,40 @@
 
 parameters:
-  # build agent pool can be overwritten on build template level 
-  pool: 'Azure-VSTS-VS2017'
-  azureSubscription: 'Reform-CFT-VH-Dev'
```

Comment:
I notice there's a `productionAzureSubscription` below, you don't need this default one?